setup.py

```diff
@@ -680,17 +733,17 @@ def summary_report(self, feature, unsafe_zlib):
 
         if not all:
             print("To add a missing option, make sure you have the required")
-            print("library, and set the corresponding ROOT variable in the")
-            print("setup.py script.")
+            print("library and headers.")
+            print("See http://pillow.readthedocs.org/en/latest/installation.html#building-from-source")
```

Comment:
That reminds me, because of the way that RTD is setup with versions, I can't find a deep permalink for the installation docs. Which means that all of the googable links to the documentation 404 eventually, unless they're going off of latest.